packages/networks/src/networks/avalanche.ts

```diff
@@ -62,9 +62,8 @@ export const avalanche: NetworkConfig = {
   startBlock: 17188332,
   subgraph: {
     endpoint:
-      'https://api.thegraph.com/subgraphs/name/unlock-protocol/avalanche',
-    endpointV2:
```

Comment:
I think the apps still use `endpointV2` though... so we need to change them as well.